config-builder/src/test/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/loader/PrivateKeyLoaderTest.java

```diff
@@ -188,9 +188,7 @@ void loadNonexistingDirectory() throws IOException {
                 "    \"keyPassword\": \"\"\n" +
                 "  }\n" +
                 "}";
-        when(fileUtils.getInputStream(any(File.class))).thenReturn(new ByteArrayInputStream(json.getBytes()));
-
-        assertThrows(BundleLoadException.class, () -> loader.load(new Bundle(), rootProjectDir.getRoot()));
+        loadPrivateKeys(json, "json", false);
```

Comment:
can you copy some of the test cases and make them fail if the keys are missing? then we test all the paths

Reply:
Adding more tests